src/diffusers/pipelines/ltx/pipeline_ltx.py

```diff
@@ -753,7 +759,25 @@ def __call__(
                 latents, self.vae.latents_mean, self.vae.latents_std, self.vae.config.scaling_factor
             )
             latents = latents.to(prompt_embeds.dtype)
-            video = self.vae.decode(latents, return_dict=False)[0]
+
+            if not self.vae.config.timestep_conditioning:
+                timestep = None
+            else:
+                noise = torch.randn(latents.shape, generator=generator, device=device, dtype=latents.dtype)
+                if not isinstance(decode_timestep, list):
+                    decode_timestep = [decode_timestep] * batch_size
+                if decode_noise_scale is None:
```

Comment:
it is strange for `None` to have a special behavior like this. can we default it to `None`?

Reply:
Yes that sounds good. I'll make the example use the recommended default value of 0.025 though as demo